### PR TITLE
Disable TVTK tests if its prerequisites aren't met

### PR DESCRIPTION
**Pull request by guyer**

### CODE REVIEW

**Comment by guyer** [comment]
Ignore the test failures. They're addressed on #721.

File: fipy/viewers/vtkViewer/vtkViewer.py [hunk 1]
```diff
@@ -15,6 +15,11 @@ def _checkForTVTK():
             from tvtk.api import tvtk
         except ImportError as e:
             from enthought.tvtk.api import tvtk
+
+        # With Python 2.7 end-of-life, bitrot has set in.  While tvtk is
+        # available from conda-forge, it can raise an exception when it
+        # internally does this import.
+        from collections.abc import Sequence
```
Comment (wd15) [inline fipy/viewers/vtkViewer/vtkViewer.py:22]: I don't fully understand why the import is required. If it fails then let it fail and the user gets a warning that it isn't available. How does the extra import impact that?

Comment (guyer) [reply fipy/viewers/vtkViewer/vtkViewer.py:22]: If we do that then running CI on Python 2.7 is pointless. We'll never get pull requests, even though we don't really care about TVTK failures and we know why they happen.

Comment (wd15) [reply fipy/viewers/vtkViewer/vtkViewer.py:22]: I get it. Tvtk always imports, but fails during the run stage so you're trying to make the failure happen during the import stage.

**Review by wd15 — APPROVED** [review]

**Review by tkphd — APPROVED** [review]

**Pull request merged**